lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,112 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # prevent autorun from running tests when the vm closes
+  def self.autorun
+  end
+end
+
+module Mutant
+  class Integration
+    # Minitest integration
+    class Minitest < self
+      # represents the data associated with a test
+      class TestCase
+        include Concord.new(:class_name, :test_method)
+        include Adamantium
+
+        attr_reader :class_name
+
+        def run_passes?(reporter)
+          ::Minitest::Runnable.run_one_method(klass, test_method, reporter)
+          reporter.passed?
+        end
+
+        def method_name
+          test_method[5..-1]
```

Comment:
What is `test_method` ? If it's a `String` then this way is probably better since `String` is not `Enumerable`. I was thinking it was an `Array` or something, in which case `#drop` is better.

Aside: I'm not really sure why they decided `String` shouldn't be `Enumerable`. I find myself wanting to treat it as a list of "things" and apply `Enumerable` methods on a semi-regular basis.